internal/log/log.go

```diff
@@ -2,9 +2,16 @@ package log
 
 import (
 	"fmt"
+
 	"github.com/logrusorgru/aurora"
 )
 
+// InitColors initialize the logger with or without
+// colors.
+func InitColors(colors bool) {
+	Au = aurora.NewAurora(colors)
```

Comment:
Not the best to have a `Au` variable that an be access global and still be null.
I've added a comment above but I'm still not happy

Reply:
Fixed by using a singleton, Still not the best, but an easy fix